packages/github-actions/actions/get-plugin-releases/src/get-plugin-releases.js

```diff
@@ -53,12 +53,14 @@ function parsePluginVersions( releases = {} ) {
 	const output = [];
 
 	if ( slug !== 'wordpress' ) {
+		const latest = releases.version;
 		const versions = Object.keys( releases.versions )
 			.filter(
 				( version ) =>
 					version !== 'trunk' &&
 					version !== 'other' &&
-					! version.includes( 'beta' )
+					! version.includes( 'beta' ) &&
+					( includeRC || semverCompare( latest, version ) <= 0 )
```

Comment:
Thanks for the additional changes. This now works when we have includeRC set to false it only returns 8.7.0 as the latest version. However if we set includeRC to true it returns both 8.8.0 and 8.8.0-rc.1
I'd be expecting it to return only the 8.8.0-rc.1, should it be using isRC instead of includeRC?

Reply:
You're right @mikkamp it should be checked with `isRC` function. I tweaked it here https://github.com/woocommerce/grow/pull/102/commits/8db7d34555ee882d08da668c96a2b5f1336e580a